src/hooks/useToast.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+const Ctx = React.createContext(null);
+
+const ToastContainer = props => (
+  <div style={{ position: "fixed", right: 0, top: 0 }} {...props} />
```

Comment:
Instead of adding css inline props we could use a `styled-components` with the current styles so we preserve the same pattern as the rest of the application